posthog/api/test/test_feature_flag.py

```diff
@@ -113,6 +114,19 @@ def test_create_feature_flag(self, mock_capture):
             },
         )
 
+    def test_legacy_create_feature_flag_with_token_override(self):
+        team2 = Organization.objects.bootstrap(self.user)[2]
+        self.client.force_login(self.user)
+
+        response = self.client.post(
+            f"/api/feature_flag/?token={team2.api_token}",
+            {"name": "Alpha feature", "key": "alpha-feature", "filters": {"groups": [{"rollout_percentage": 50}]}},
+            format="json",
+        )
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        instance = FeatureFlag.objects.get(id=response.json()["id"])
+        self.assertEqual(instance.team, team2)
```

Comment:
Do we have some tests for the non-legacy way?

Reply:
Well I didn't touch the new stuff and there were tests for it before. Can add an extra one though

Reply:
Which should override? We currently have it in `master` such that a token passed in as data will override the URL one, which is probably incorrect?

Reply:
Basically I mean a test that'd be a working example of the project-nesting-compatible way of using `token` in the API (unless there already is one and I just missed it). Regarding overrides, hm, IMO the API should be structured in a way that makes it impossible to specify both `id` and `api_token`, as that can only result in a confusing conflict.